assets/queries/openAPI/encoding_header_content_type_improperly_defined/query.rego

```diff
@@ -0,0 +1,71 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	header := doc.paths[path][operation].responses[r].content[c].encoding[e].headers[h]
```

Comment:
I think `.content[c].encoding[e].headers[h]` is duplicated in too many lines... Shouldn't be better create a function `check_content_header` and make checks inside this function?